### ISSUE TITLE
[BUG] GUI doesn’t have option for white space in Individual entity

### ISSUE THREAD

**Issue by tramperb** [flixlix/power-flow-card-plus#596]
Not a big issue but ...
V0.2.2 added the option to set ' unit_white_space: ' for Indicidual entities.

The option (default is false) _can_ be set to true in YAML (and it works) but is still not available in the GUI.

Thanks for the work on the card - it's working well and is a key component in managing efficient energy consumption.
(BTW  the readme still says v0.2.0 is out now)

**Comment by flixlix** [comment]
Can you try [clearing your cache](https://how-can-i.luca-felix.com/clear-cache)?

In the readme the alert is correct, because it was from version 0.2.0 on that the biggest changes were made

**Comment by tramperb** [comment]
Yeah, did that.  Restarted HA, restarted the PC, cleared cache on PC, same result on both.  Used Chrome and Firefox (which had never been used for HA) and _still_ no white space option in the GUI.  On Android via the app is the same.
All current versions : HA docker container 2024.4.3, frontend 20240404.2, HACS 1.34.0, Power Flow Card Plus v0.2.2

**Comment by flixlix** [comment]
Thank you, it seems like for some reason this change didn't make it to the final build. Here is the correct file if you want to install it manually, otherwise the change should take effect in the next version:

[Uploading power-flow-card-plus.js.zip…]()

**Comment by tramperb** [comment]
Thanks.  I'll wait for the next version (since I have it in yaml anyway) - unless you want me to test it.

**Comment by flixlix** [comment]
should be fixed now
